Frameworks/D2W/ERModernDirectToWeb/Components/Nonlocalized.lproj/ERMD2WEditToOneTypeAhead.wo/ERMD2WEditToOneTypeAhead.wod

```diff
@@ -1,3 +1,8 @@
+ERMD2WEditToOneTypeAhead : AjaxUpdateContainer {
+  id = updateContainerID;
+  class = "ToOneTypeAheadFieldUC";
```

Comment:
As I see it you change the CSS class from *ERMD2WEditToOneTypeAhead* to *ToOneTypeAheadFieldUC*. Is that intentional?